android/app/src/main/java/com/mulberry/ody/data/remote/core/entity/meeting/mapper/MeetingRequestMapper.kt

```diff
@@ -2,19 +2,25 @@ package com.mulberry.ody.data.remote.core.entity.meeting.mapper
 
 import com.mulberry.ody.data.remote.core.entity.meeting.request.MeetingRequest
 import com.mulberry.ody.data.remote.core.entity.meeting.request.NudgeRequest
+import com.mulberry.ody.domain.model.Address
 import com.mulberry.ody.domain.model.MeetingCreationInfo
 import com.mulberry.ody.domain.model.Nudge
 
 fun MeetingCreationInfo.toMeetingRequest(): MeetingRequest =
     MeetingRequest(
         name = name,
-        date = date,
-        time = time,
-        targetAddress = targetAddress,
-        targetLatitude = targetLatitude,
-        targetLongitude = targetLongitude,
+        date = dateTime.toLocalDate().toString(),
+        time = dateTime.toLocalTime().toString(),
+        targetPlaceName = destinationAddress.toAddressString(),
+        targetLatitude = destinationAddress.latitude,
+        targetLongitude = destinationAddress.longitude,
     )
 
+private fun Address.toAddressString(): String {
+    if (placeName.isBlank()) return detailAddress
+    return "$detailAddress ($placeName)"
```

Comment:
유저 피드백 중에 도로명 주소가 어디인지 모르겠다는 피드백이 생각보다 많았어요.

상호명으로 전반적인 대체가 불가한 상황이니 대체 가능한 상황에 대해서는 상호명만을 지원하는 건 어떨까요? 다만 출발지는 도로명 주소이고 도착지는 상호명일 때 일관성적인 측면에서 우려되는 부분은 있어요.

그럼에도 개인적으로는 이익이 크다고 생각합니다.
도로명 주소는 최소8-10자로 구성되는데요. 만약 상호명이 길다면 도로명 주소(상호명)이 두 줄을 넘어갈 수도 있다는 생각이에요.

결론적으로는 상호명으로 대체 가능한 상황에는 상호명 만을 띄워주자는 의견입니다.
```suggestion
    return "$placeName"
```

의견에 대한 수용 여부는 올리브가 담당자이니 올리브 선택에 맡길게요! 다만 그 이유를 답글로 남겨주면 기쁠 것 같아요 😄 

- 일관적이지 않게 보일 수 있다 :

Reply:
맞습니다. 제가 pr 본문에 첨부한 로그 화면에서도 주소가 두 줄이 넘어가서, 보기에도 예쁘진 않았어요.
저도 콜리가 말한 방식을 생각했었는데, `상세 주소(상호명)`으로 구현한 이유는 아래와 같습니다.

<br>

<img src="https://github.com/user-attachments/assets/2d1bc1ec-91c6-4472-8aa0-f8763b781df5" height="200"/> 

<img src="https://github.com/user-attachments/assets/ba0ee980-a830-4838-b631-2feaba9b8073" height="200"/> 

1. 직접 화면 상에서 봤을 때, 일관적이지 않아 어색하다고 느꼈어요. 사용자는 어떤 기준으로 상세 주소와 상호명이 구분되는지 모르기 때문에, 혼란을 겪을 수도 있다고 생각합니다.
2. 어딘가에는 상세 주소를 제공해야 한다고 생각해요. 특히 가게나 건물은 동일한 이름이 꽤 많기 때문에, 사용자가 헷갈릴 수도 있을 것 같아요. `메인 화면`에서는 `상호명`만을 보여주고, `로그 화면`에서는 `상세 주소`까지 보여주면 좋을 것 같은데 현재는 그렇게 문자열을 분리할 수 있는 구조가 아니기도 하고요. (상세 이유는 PR 본문 참고)

그런데 저도 콜리 의견에 동의합니다. 이 부분은 데모데이 전에 다른 팀원들에게도 공유해서 의견을 하나로 모으면 좋을 것 같아요. 일단 이대로 PR merge하고 추후 다시 얘기 꺼내볼게요~